### DISCUSSION TITLE
피드 작성 시 멤버 태그 지원 문의

### DISCUSSION THREAD

**Discussion by hanaaple** [naverz/zepeto-studio-kor#683] [5. Q&A]
혹시 제페토 월드 스크립트 내에서 피드 작성시 멤버(캐릭터) 태그 기능은 업데이트 예정에 있을까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 해당 기능을 제공할 예정은 있으나 단기간 내에는 업데이트가 힘들 것 같습니다. 
양해 부탁 드립니다.
감사합니다.
